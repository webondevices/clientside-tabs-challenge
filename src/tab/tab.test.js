import React from "react";
import renderer from "react-test-renderer";
import Tab from "./tab";

it("renders correctly", () => {
  const tree = renderer.create(<Tab title="UK News" id="uk-news" />).toJSON();
  expect(tree).toMatchSnapshot();
});
