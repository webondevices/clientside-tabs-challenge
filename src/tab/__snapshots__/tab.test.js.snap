// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <a
    className="tab"
    href="#uk-news"
    id="uk-news"
  >
    UK News
  </a>,
  <div
    className="tab-panel-container"
  />,
]
`;
